plugins/pipelines/listEvents.js

```diff
@@ -44,7 +44,7 @@ module.exports = () => ({
                     }
 
                     const eventType = request.query.type || 'pipeline';
-                    const config = { params: { type: eventType } };
+                    const config = { params: { type: eventType }, sort };
```

Comment:
What happens when `sort` is not specified in the `request.query`?
i.e. when `config.sort: undefined`

Should we add `sort` to `config` only when it is explicitly specified?